src/output_parser/sarif_vulnerability_mapping.csv

```diff
@@ -53,7 +53,8 @@ oyente,Arithmetic_12,Integer Underflow.,Arithmetic
 oyente,AccessControl_16,Parity Multisig Bug 2.,AccessControl
 oyente,Reentrancy_6,Re-Entrancy Vulnerability.,Reentrancy
 oyente,TimeManipulation_3,Timestamp Dependency.,TimeManipulation
-securify,Reentrancy_7,DAO,Reentrancy
+oyente,FrontRunning_6,Transaction-Ordering Dependency.,FrontRunning
+Dsecurify,Reentrancy_7,DAO,Reentrancy
```

Comment:
I think there is a D at the beginning of the line that should not be there

Reply:
Well observed, apparently a character that went to the wrong window. I pushed the fix. Moreover, I added another test exercising the various flag, see `tests/coverage.sh`. Moreover, there were still some path inconsistencies regarding whether the pathes are relative to the current dir or to smartbugs' home dir. As it is now, it is possible to install smartbugs globally, and all files written by smartbugs are put into the user space (either below the current working dir or into `$HOME/.solcx`).

Please recheck again ...